config/bootstrap.php

```diff
@@ -14,6 +14,11 @@
 defined('MUNEE_CACHE') || define('MUNEE_CACHE', MUNEE_FOLDER . DS . 'cache');
 // Define default character encoding
 defined('MUNEE_CHARACTER_ENCODING') || define('MUNEE_CHARACTER_ENCODING', 'UTF-8');
+// Are we using Munee with URL Rewrite (.htaccess file)?
+$requestUri = isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : '';
+defined('MUNEE_USING_URL_REWRITE') || define('MUNEE_USING_URL_REWRITE', strpos($requestUri, 'files=') === false);
```

Comment:
I dunno...I kind of like having 3 less characters haha.  It's like this in other parts of the file, so I'll keep it.